.github/workflows/build-ci-container.yml

```diff
@@ -18,95 +18,70 @@ on:
       - '.github/workflows/containers/github-action-ci/**'
 
 jobs:
-  # TODO(boomanaiden154): Switch this back to a single stage build when we can
-  # run this on the self-hosted runners and don't have to do it this way to
-  # avoid timeouts.
-  build-ci-container-stage1:
+  build-ci-container:
     if: github.repository_owner == 'llvm'
-    runs-on: ubuntu-latest
+    runs-on: depot-ubuntu-24.04-16
+    outputs:
+      container-name: ${{ steps.vars.outputs.container-name }}
```

Comment:
Why do we need to set these as outputs?

Reply:
They're used in the push-ci-container job.